Documentation/common-issues.md

```diff
@@ -383,3 +385,34 @@ Uncordon the node:
 ```
 kubectl uncordon <node-name>
 ```
+
+# Rook Agent modprobe exec format error
+## Symptoms
+* PersistentVolumes from Ceph fail/timeout to mount
+* Rook Agent logs contain `modinfo: ERROR: could not get modinfo from 'rbd': Exec format error` lines
+
+## Solution
+If it is feasible to upgrade your kernel, you should upgrade to `4.x`, even better is >= `4.7` due to a feature for CephFS added to the kernel.
+
+If you are unable to upgrade the kernel, you need to go to each host that will consume storage and run:
+```
+modprobe rbd
+```
+This command inserts the `rbd` module into the kernel.
+
+To persist this fix, you need to add the `rbd` kernel module to either `/etc/modprobe.d/` or `/etc/modules-load.d/`.
+For both paths create a file called `rbd.conf` with the following content:
+```
+rbd
+```
+Now when a host is restarted, the module should be loaded automatically.
+
+# Using multiple shared filesystem (CephFS) is attempted on a kernel version older than 4.7
+## Symptons
+* Mounting a filesystem other than the first created, will cause the first filesystem to be always mounted instead
+
+## Solution
+The only solution to this problem is to upgrade your kernel to `4.7` or higher.
+This is due to a mount flag added in the kernel version `4.7` which allows to chose the filesystem by name.
+
+For additional info on the kernel version requirement for multiple shared filesystems (CephFS), see [Filesystem - Kernel version requirement](filesystem.html#kernel-version-requirement).
```

Comment:
this link is broken since it is using `.html`.  it needs to be `.md` here and the post processing to generate the docs website will convert it to html.  An example from earlier in this document: `[Flexvolume configuration pre-reqs](./k8s-pre-reqs.md#flexvolume-configuration)`